packages/nimiq-css/src/css/utilities.css

```diff
@@ -148,6 +148,7 @@
   background-image: radial-gradient(at 100% 100% in oklab, var(--nq-gradient-from) 0%, var(--nq-gradient-to) 100%);
   --transitions: --nq-gradient-from 200ms var(--nq-ease), --nq-gradient-to 200ms var(--nq-ease);
   transition: var(--transitions);
+  margin: 0.5rem 0.25rem;
```

Comment:
I don't like adding this margin.

This change will break a LOT of things

We use this utility in many places, and this change will just break a lot of things.

What exactly are you trying to solve?